### PR TITLE
create miner doc

### PR DESCRIPTION
**Pull request by schomatis**
Rendered [here](https://github.com/filecoin-project/lotus/blob/schomatis/feat/docs/create-miner/documentation/en/dev/create-miner.md).

-----------------------------

@arajasek This is just a complementary doc to the main one so don't feel obligated to sink much time into it (although feel free to change as much as you want), just a basic proof reading and link checking would be fine, also please check that the knowledge this document already assumes from the main architecture document is actually there.

### CODE REVIEW

File: documentation/en/dev/create-miner.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+The `lotus-storage-miner init` command option creates a new storage miner node. We will only be able to run the command once the chain has been synced by the full node (which needs to be running) and it will also require the download of the [proof parameters](https://filecoin.io/blog/filecoin-proof-system/) (of several GBs, so it may take some time).
+
+The main options that define a miner are the owner and worker addresses associated to it (stored in [`MinerInfo`](https://github.com/filecoin-project/specs-actors/blob/master/actors/builtin/miner/miner_state.go), a substructure of the Miner Actor State) and its peer ID. We use default values for all of these options in the command and briefly described them here:
```
Comment (arajasek) [inline documentation/en/dev/create-miner.md:50]: described -> describe

File: documentation/en/dev/create-miner.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+* Owner: address of the account actor that manages this miner (receives its payments, can change its information, etc.). By default this option is set to the default wallet address (and associated account actor), in the case of this example it will be the address just created with the `wallet new` command.
+
+* Worker: address of an account actor that signs blocks and submits proofs associated with this miner, it can be the same as the owner address. By default the command will create a new wallet address for it, of type BLS (the only one accepted for worker addresses).
+
+* [Peer ID](https://docs.libp2p.io/reference/glossary/#peerid): a network ID (belonging to the `libp2p` stack) used to contact the miner directly off-chain (e.g., to make a storage deal). Note the difference with the rest of the communication in the Filecoin network that happens largely inside the chain itself: when we "send" messages to the different actors that is actually a VM abstraction meaning we execute the method in the VM itself run by logic of the targeted actor, physically (at the network TCP/IP level) we broadcast the message to all of our peers to be included in a Filecoin block.
```
Comment (arajasek) [inline documentation/en/dev/create-miner.md:56]: Split the sentence starting "Note" into 2-3 sentences

File: documentation/en/dev/create-miner.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+There is a one-to-one mapping between the pair of actor and method number (`To:`/`Method:` fields) in a message in the VM, and the Go function in an actor's exported methods list that implement it. In this case, for the Power Actor list of method numbers defined in [`MethodsPower`](https://github.com/filecoin-project/specs-actors/blob/master/actors/builtin/methods.go), the `CreateMiner` method number 2 will correspond to the Go function with the same index in the list of methods returned by [`Exports()`](https://github.com/filecoin-project/specs-actors/blob/master/actors/builtin/power/power_actor.go) (and normally also the same name, here `(Actor).CreateMiner()`).
+
+The Power Actor in `CreateMiner()` will do two things:
+
+1. Send *another* message, `Exec`, to the Init Actor to instruct it to create the miner actor with the information provided by `lotus-storage-miner` and receive its ID address (this ID is the one returned to the CLI command).
```
Comment (arajasek) [inline documentation/en/dev/create-miner.md:78]: wanna specify that this is an "internal" message?

**Comment by schomatis** [comment]
@arajasek Thanks for reviewing, please apply any corrections you deem necessary, I trust you to change it in whatever way you see best.

**Comment by schomatis** [comment]
Merging as is to get something in, we can improve on this later if it proves to be useful.

**Pull request merged**